### PR TITLE
Mobile 3320

### PR DESCRIPTION
**Pull request by crazyserver**

### CODE REVIEW

**Review by dpalou — APPROVED** [review]

**Comment by dpalou** [comment]
Hi Pau,

the changes look good to me, but it seems some E2E tests are failing. Maybe they only look for text in aria-label and not in title?

**Comment by crazyserver** [comment]
Thanks Dani! Can you have a look again?

File: src/core/features/comments/pages/viewer/viewer.page.ts [hunk 1]
```diff
@@ -262,48 +275,65 @@ export class CoreCommentsViewerPage implements OnInit, OnDestroy {
     }
 
     /**
-     * Add a new comment to the list.
+     * Send the comment or store it offline.
      *
-     * @param e Event.
+     * @param text Comment to add
+      Event.
```
Comment (dpalou) [inline src/core/features/comments/pages/viewer/viewer.page.ts:281]: This line should be removed.

File: src/core/features/comments/pages/viewer/viewer.page.ts [hunk 2]
```diff
@@ -160,9 +164,15 @@ export class CoreCommentsViewerPage implements OnInit, OnDestroy {
                     commentsResponse.comments.length >= CoreCommentsProvider.pageSize;
             }
 
-            comments = await Promise.all(comments.map((comment) => this.loadCommentProfile(comment)));
+            this.comments = comments.concat(this.comments);
```
Comment (dpalou) [inline src/core/features/comments/pages/viewer/viewer.page.ts:167]: This change means that we will display comments without user data, and then the user data will be loaded. Wouldn't it be better to wait for all data to be loaded before displaying them, as it was done before?

**Review by dpalou — CHANGES_REQUESTED** [review]

**Comment by crazyserver** [comment]
Thanks Dani! Solved! I also changed the way it scrolls to bottom to avoid scrolling when the page is partially loaded.

**Comment by crazyserver** [comment]
Also added a new commit to prevent new line from being shown before submitting from send message form

**Review by dpalou — APPROVED** [review]

**Pull request merged**